### PR TITLE
Feature/update pr comments

### PR DESCRIPTION
**Pull request by SathyaShreeRajkumar**
- Resolved PR comments
- Removed .vs
- Updated testcase for controller

### CODE REVIEW

File: AirlineReservation.Tests/Test/Controller/AirlineControllerTest.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            mockAirlinesService.Verify(service => service.getAllAirlines(), Times.Once);
+
+            result
+                .Should().BeOfType<OkObjectResult>().Which.Value.Should()
+                .BeOfType<List<AirlineModel>>().And.Subject.As<List<AirlineModel>>()
+                .Should().BeEquivalentTo(mockAirlines);
+
+            result
+                .Should().BeOfType<OkObjectResult>().Which.StatusCode.Should()
+                .Be(StatusCodes.Status200OK);
```
Comment (aashikvilla) [inline AirlineReservation.Tests/Test/Controller/AirlineControllerTest.cs]: OkObjectResult by definition has a status code of 200, This feels redundant

File: AirlineReservation/Controllers/AirlineController.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+        public AirlineController(IAirlineService airlineService)
+        {
+            _airlineService = airlineService;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> GetAllAirlines()
+        {
+            var airlines = await _airlineService.getAllAirlines();
```
Comment (aashikvilla) [inline AirlineReservation/Controllers/AirlineController.cs]: use Pascal casing for methods

File: AirlineReservation/Services/Airline/AirlineService.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+namespace AirlineReservation.Services.Airline
+{
+    public class AirlineService : IAirlineService
+    {
+        private IMapper _mapper;
+        private IDatabaseContext _databaseContext;
+
+        public AirlineService(IMapper mapper, IDatabaseContext databaseContext)
+        {
+            _mapper = mapper;
```
Comment (aashikvilla) [inline AirlineReservation/Services/Airline/AirlineService.cs]: are we using mapper here?

File: AirlineReservation/Services/Airline/AirlineService.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+            _databaseContext = databaseContext;
+        }
+        public async Task<List<AirlineModel>> getAllAirlines()
+        {
+            return await _databaseContext.Airlines.Find(airline => true).ToListAsync();
```
Comment (aashikvilla) [inline AirlineReservation/Services/Airline/AirlineService.cs:17]: Maybe for later, have a look at projections in mongoDb when you get time

File: AirlineReservation.Tests/Test/Controller/AirlineControllerTest.cs (hunk 1 above)
Comment (SathyaShreeRajkumar) [reply AirlineReservation.Tests/Test/Controller/AirlineControllerTest.cs]: Checked with OkObjectResult.

File: AirlineReservation/Services/Airline/AirlineService.cs (hunk 3 above)
Comment (SathyaShreeRajkumar) [reply AirlineReservation/Services/Airline/AirlineService.cs]: Meant to use in upcoming methods. Removed for now.

File: AirlineReservation/Services/Airline/AirlineService.cs (hunk 4 above)
Comment (SathyaShreeRajkumar) [reply AirlineReservation/Services/Airline/AirlineService.cs:17]: sure

File: AirlineReservation.Tests/Test/Controller/AirlineControllerTest.cs [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+            var result = await airlineController.GetAllAirlines();
+
+            mockAirlinesService.Verify(service => service.GetAllAirlines(), Times.Once);
+
+            result
+                .Should().BeOfType<OkObjectResult>().Which.Value.Should()
+                .BeOfType<List<AirlineModel>>().And.Subject.As<List<AirlineModel>>()
+                .Should().BeEquivalentTo(mockAirlines);
+
+            result.Should().BeOfType<OkObjectResult>();
```
Comment (kool7) [inline AirlineReservation.Tests/Test/Controller/AirlineControllerTest.cs:33]: This is not needed we are already checking type in above assert statement.

File: AirlineReservation/Models/Data/AirlineModel.cs [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+namespace AirlineReservation.Models.Data
+{
+    public record AirlineModel
+    {
+        [BsonId]
+        [BsonRepresentation(BsonType.ObjectId)]
+        public string AirlineId { get; set; }
+
+        [BsonElement("name")]
```
Comment (kool7) [inline AirlineReservation/Models/Data/AirlineModel.cs:12]: use convention registry to rename POCO properties to camel case.

**Pull request merged**